sequence/arguments/api.graphql

```diff
@@ -57,23 +60,21 @@ type Query {
             }
             """
         )
-    # A sequence call that sequences converCityOrLocaleToLatLong and weather
-    # this allows for a cityOrLocale argument to be used to fetch weather data by lat/lon
-    # we have to use an arguments setting for the second query, because   
-    # query arguments only flow to the first query in the sequence
-    # but we need units to flow to the second query
-    # we are saying that the argument named units in the query weather 
-    # gets its value from the argument units in the enclosing query
-    # In addition to the arguments explictly passed down to the second query,
-    # it also gets the output of the first query
-    # in this case, the first query returns a type Coord, and therefore its fields lat and lon are
-    # available to the second query.
+    # Below is a sequence call that sequences `convertCityOrLocaleToLatLong` and `weather`.
+    # Now the `cityOrLocale` argument can fetch weather data by lat/lon.
+    # We have to use an `arguments` setting for the second query, because   
+    # enclosing query arguments are only directly accessible from the first query in the sequence,
+    # but we need `units` in the second query.
+    # The argument named `units` in the query `weather` 
+    # gets its value from the argument `units` in the enclosing query
+    # The second query receives the arguments passed down to it,
```

Comment:
Maybe more explicit about what is going on?

A step's field arguments can be set from:
 - fields returned by any previous step
 - the sequence field's arguments.

For an argument `a` without an arguments entry for a argument in step it defaults to the closest previous step that has a field named `a`.
This can be overridden with these entries in arguments:

- `{name:"a" field: "b" } - set argument `a` from the closest previous step that has field `b`.
- `{name:"a" argument: "c" }  - set argument `a` from the sequence field argument `c`

Then maybe expand/be explicit with the current example

Reply:
I agree a full-fledged example could be helpful, so I've linked out to the docs here.